packages/strapi-plugin-upload/config/functions/bootstrap.js

```diff
@@ -51,6 +51,30 @@ module.exports = async cb => {
           });
 
           await pluginStore.set({key: 'provider', value});
+
+          const dirPath = `${strapi.config.appPath}/public/uploads`;
+   
+          try {
+            // Check if directory exists
+            await new Promise((resolve, reject) => fs.exists(dirPath, (exists) => {
```

Comment:
Please do not use `fs.exists`, the method is deprecated (see https://nodejs.org/api/fs.html#fs_fs_exists_path_callback). You should use `fs.access` instead.